include/BidirectionalCoroutine.hpp

```diff
@@ -142,8 +231,9 @@ namespace com {
 					
 					template<class F, typename std::enable_if<!std::is_void<RType<F&,Yield&>>::value && !std::is_assignable<RType<F&, Yield&>, R>::value,int>::type =0>
 					static void apply(Yield & yield, F & f) {
-						static_assert(std::is_void<RType<F&,Yield&>>::value || std::is_assignable<RType<F&, Yield&>, R>::value,"return type of f must be void, or assignable to the return type of this coroutine");
-						f(yield); // This will never execute.
+						static_assert(	std::is_void<RType<F&,Yield&>>::value || std::is_assignable<RType<F&, Yield&>, R>::value,
```

Comment:
Wasn't there a [[noreturn]] for this somewhere?  I don't see it now.

Reply:
GCC was warning that it was being ignored, I think I had it in the wrong place. Going to test again momentarily.

Reply:
Reintroduced (in the correct position) by 85964d3.